### ISSUE TITLE
Add more dungeons (New dungeons mod)

### ISSUE THREAD

**Issue by PaintNinja** [PaintNinja/CraftMine#12]
Might be a good idea to take a look at New Dungeons originally by Slidedrum, now maintained by GoToLink (really interested in this one, although it doesn't add much): http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1281102-1-5-2-1-4-4-1-4-5-new-dungeons-v1-8-5-ssp-smp, http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1287599-gotolink-updates-formivores-options-2015-08-01#NewDungeons

See issue #9 for more details.

**Comment by PaintNinja** [comment]
Tried it out and it loaded up fine first time. It took quite a long time for me to find a dungeon from this mod but once I did I liked it. It's a lot like the rare jungle dungeons from MC1.3, but underground.

Known issues are that sometimes the tripwires fail to place and sometimes the dispensers face the wrong way so they can't fire properly. Also the amount of mob spawners and stuff inside is a bit too high. We need to look into this a bit more...
